internal/testutil/acc/shared_resource.go

```diff
@@ -166,3 +169,11 @@ func NextProjectIDClusterName(totalNodeCount int, projectCreator func(string) st
 	}
 	return project.id, RandomClusterName()
 }
+
+// setupTestsSDKv2ToTPF sets the Preview environment variable to false so the previous version in migration tests uses SDKv2.
+// However the current version will use TPF as the variable was read when it was true.
```

Comment:
This is a bit surprising to me, running a local test just to double-check

Reply:
sure, keep me posted

Reply:
Aha. I thought we re-read it when the provider is configured.

```suggestion
// However the current version will use TPF as the variable is only read once during import when it was true.
```